features/mbedtls/platform/COMPONENT_PSA_SRV_IMPL/inc/default_random_seed.h

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
Is this the final location we should place the file? @alzix is working on adding a new `TARGET_PSA` folder mechanism. I think we should place these files into a `TARGET_PSA` folder (perhaps `features/mbedtls/platform/TARGET_PSA`). Sorry for the late design change, but `TARGET_PSA` is a late design change that affects this PR.

Ensure you are aligned with https://github.com/ARMmbed/mbed-os/pull/8723 so we don't have too many disparate folders for what should all be the same folder.